quinn-udp/tests/tests.rs

```diff
@@ -87,6 +87,30 @@ fn ecn_v4() {
     }
 }
 
+#[test]
+#[cfg_attr(not(target_os = "linux"), ignore)]
+fn gso() {
+    let send = UdpSocket::bind("[::1]:0").unwrap();
```

Comment:
This will fail if IPv6 support is disabled at OS level, i can modify this and basic test to fallback to Ipv4 loopback in case we fail to create the Ipv6 socket.